CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Changed
 
+* Tarantool versions 2.{4,5,6} have reached EOL and removed from regression
+  testing.
```

Comment:
It's worth to mention that min/max commands won't work for old 1.10 versions anymore.

Reply:
And it's a bit inconsistent with select that still works for such versions.

Reply:
How one is relevant to another?